Source/DafnyLanguageServer.Test/Synchronization/DiagnosticsTest.cs

```diff
@@ -21,6 +21,42 @@ namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Synchronization {
   [TestClass]
   public class DiagnosticsTest : ClientBasedLanguageServerTest {
 
+    [TestMethod]
+    public async Task CrashOfLanguageServer() {
+      // git-issue-3062
+      var source = @"
+function bullspec(s:seq<nat>, u:seq<nat>): (r: nat)
+  requires |s| > 0
+  requires |u| > 0
+  requires |s| == |u|
+  ensures forall i | i < r :: s[i] != u[i]
+  ensures r == |s| || s[r] == u[r]
+{
+  if |s| == 0 then 0 else
+  if |s| == 1 then
+    if s[0]==u[0] 
+    then 1 else 0
+  else
+    if s[0] != u[0] 
+    then bullspec(s[1..],u[1..])
+    else 1+bullspec(s[1..],u[1..])
+}".TrimStart();
+      var documentItem = CreateTestDocument(source);
+      await client.OpenDocumentAndWaitAsync(documentItem, CancellationToken);
+      // Uncomment what you need.
```

Comment:
What is this comment for?

Reply:
Oh that's a placeholder generated by my script to investigate crashes in the LSP. I forgot to remove it.